src/gmm_utils.erl

```diff
@@ -97,3 +99,25 @@ convert_microseconds_to_iso_8601(TotalMicroseconds) when TotalMicroseconds >= 0
     %% compose final result
     TimeString = "PT" ++ HoursString ++ MinutesString ++ SecondsString,
     list_to_binary(TimeString).
+
+char_to_boolean($0) -> false;
+char_to_boolean($1) -> true.
+
+boolean_to_char(true) -> $1;
+boolean_to_char(false) -> $0.
+
+-type permissions() :: <<_:5*8>>.
```

Comment:
Definicja typu jest publiczna? Jeśli tak, to można by jej używać w całym serwerze

Reply:
Mogła by być. Zrobię z tego issue na Jirze, aktualnie nie mam za bardzo czasu przekopywać całego kodu w poszukiwaniu uprawnień

Reply:
Ok